js/main.js

```diff
@@ -0,0 +1,48 @@
+$( document ).ready(function(){
+
+  //COLLABORATION 
+
+  //slider show more show less
+
+		//What is it? 
+		$('.expand a').click(function(e) {
+      // "this" refers to the "a" that was clicked
+      var commonContainer = $(this).closest('.container');
+      // commonContainer now refers to the parent div that has a class of "container" on it. This commonContainer is common ancestor to the <a> tag that was clicked on, but also to the paragraph of text and the Read less link.
+     
+      // Doing this code would call slideDown() on ALL elements with a class of content and collapse. We only wanna slideDown() those that are a child of commonContainer
+      // $('.content, .collapse').slideDown();
+      
+      // So instead we do this:
+      // Using commonContainer as a starting point, we find its children using .find() that have the matching selectors "content" or "collapse", and then on those we call slideDown()
+      commonContainer.find('.content, .collapse').slideDown();
+     
+		  e.preventDefault();
```

Comment:
It doesn't _actually_ matter but usually when people do a `e.preventDefault();` its usually the first line within the function. So move this to above line 10 before the `var commonContainer = $(this).closest('.container');`